wolfcrypt/src/wc_port.c

```diff
@@ -695,6 +746,10 @@ void wc_ReadDirClose(ReadDirCtx* ctx)
         FindClose(ctx->hFind);
         ctx->hFind = INVALID_HANDLE_VALUE;
     }
+
+#elif defined(INTIME_RTOS)
+	FindRtFileClose(&ctx->FindFileData);
```

Comment:
I can see a case where FindFileData is used without initialization. Perhaps this can be zero'd in `wc_ReadDirFirst`?

Reply:
Changed setting name to zero to the whole context.